src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceService.kt

```diff
@@ -201,6 +206,46 @@ class LicenceService(
 
     recordLicenceEventForStatus(licenceId, updatedLicence, request)
     auditStatusChange(updatedLicence, request.username, request.fullName)
+
+    // if the request status is to make the licence active - send domain event
+    if (!isCurrentLicenceVariationApproved && request.status == ACTIVE) {
+      domainEventsService.recordDomainEvent(
+        LicenceDomainEventType.LICENCE_ACTIVATED,
+        updatedLicence.id.toString(),
+        updatedLicence.crn,
+        updatedLicence.nomsId,
+      )
+    }
+
+    // if the current licence is a variation approved and the request status is active - send domain event
+    if (isCurrentLicenceVariationApproved && request.status == ACTIVE) {
```

Comment:
if you extracted out the helper method as described above you could use it here too?

(this is making the existing method very large in a very large class and there's a lot of boiler plate/duplication - in situations like this it's worth thinking of refactoring techniques that can help address this)